src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -304,8 +304,11 @@ private String getPendingOrders() throws Exception {
 
 		// Walk over each line of the response.
 		String line;
-		while ( ( line = in.readLine() ) != null ) {
-			buffer.append( line );
+		try {

```

Comment:
if `in` is null here, there's no point in continuing. That means an exception was caught, would be better to return in the exception above's `catch` clause. Simply `return "[]"`